Server/serverSlate.go

```diff
@@ -67,18 +118,13 @@ func runOsCommand(cmd *exec.Cmd) int {
 	return 1
 }
 
-func simulateKeyPress(key string, keyPressCount int, timeStamps []string) {
-	timeStamps[0] = strconv.Itoa(int(time.Now().UnixNano() / 1000000))
+func simulateKeyPress(key string, keyPressCount int, timestamps []int64) {
+	timestamps[0] = time.Now().UnixNano() / 1000000
```

Comment:
Can you store this in a different variable and return it as a second value?  This would clean up the iteration code in the calculateLag function too.